packages/react-i18n/README.md

```diff
@@ -335,6 +336,56 @@ i18n.translate('MyComponent.searchResult', {
 });
 ```
 
+###### Ordinal Pluralization
+
+Ordinal pluralization lookups rely on the keys being nested under an `ordinal` key (i.e., being in an [ordinal pluralization context](https://github.com/Shopify/quilt/blob/main/packages/react-i18n/docs/react_i18n_schema.md#ordinal-pluralization)), and use the value of the `ordinal` replacement to choose the correct string:
+
+```ts
+// Assuming a dictionary like:
+{
+  "MyComponent": {
+    "searchResult": {
+      "ordinal": {
+        "one": "This is the {ordinal}st widget found",
+        "two": "This is the {ordinal}nd widget found",
+        "few": "This is the {ordinal}rd widget found",
+        "other": "This is the {ordinal}th widget found",
+      }
+    }
+  }
+}
+
+i18n.translate('MyComponent.searchResult', {ordinal: searchResultOrdinal});
+```
+
+By default, `{ordinal}` will be automatically formatted as a number. If you want to format the variable differently, you can simply pass it in another variable.
+
+```ts
+// Assuming a dictionary like:
+{
+  "MyComponent": {
+    "searchResult": {
+      "ordinal": {
+        "one": "This is the {formattedOrdinal}st widget found",
+        "two": "This is the {formattedOrdinal}nd widget found",
+        "few": "This is the {formattedOrdinal}rd widget found",
+        "other": "This is the {formattedOrdinal}th widget found",
+      }
+    }
+  }
+}
+
+i18n.translate('MyComponent.searchResult', {
+  ordinal: searchResultOrdinal,
+  formattedOrdinal: i18n.formatNumber(searchResultOrdinal),
```

Comment:
Since `translate` already formats the `ordinal` value as a number, I suggest using a different example, such as applying some visual formatting:

```suggestion
  formattedOrdinal: <b>{searchResultOrdinal}</b>,
```